insights/parsers/bond.py

```diff
@@ -38,22 +38,22 @@
 ``/proc/net/bonding``.
 
 Examples:
-    >>> bond_info = shared[Bond]
+    >>> type(bond_info)
+    <class 'insights.parsers.bond.Bond'>
     >>> bond_info.bond_mode
-    '0'
+    '4'
     >>> bond_info.partner_mac_address
-    None
+    '00:00:00:00:00:00'
     >>> bond_info.slave_interface
-    ['eno1', 'eno2']
+    ['eth1', 'eth2']
     >>> bond_info.aggregator_id
-    ['3', '2', '3']
+    ['3', '3', '2']
     >>> bond_info.xmit_hash_policy
-    layer2
-    >>> bond_info.active_slave
-    enp17s0f0
+    'layer2'
```

Comment:
Why did you remove the check of `bond_info.active_slave`? Better keep it here

Reply:
@xiangce - Test data mode type is different (its not active-backup mode so no active_slaves)

Reply:
Done